src/NodaTime.Test/DurationTest.Construction.cs

```diff
@@ -238,6 +256,20 @@ public void FactoryMethods_OutOfRange()
             
             // No such concept as BigInteger.Min/MaxValue, so use the values we know to be just outside valid bounds.
             AssertOutOfRange(Duration.FromNanoseconds, Duration.MinNanoseconds - 1, Duration.MaxNanoseconds + 1);
+
+            void AssertLimitsInt32(Func<int, Duration> factoryMethod, int[] allCases) =>
```

Comment:
Any particular reason to require the `Int32`/`Int64` suffix? Overload resolution will pick the right one, won't it? (and it doesn't affect the choice of parameters other than the type.)